pilot/pkg/model/config_test.go

```diff
@@ -177,42 +178,82 @@ func TestServiceKey(t *testing.T) {
 func TestSubsetKey(t *testing.T) {
 	hostname := model.Hostname("hostname")
 	cases := []struct {
-		hostname model.Hostname
-		subset   string
-		port     *model.Port
-		want     string
+		gotOverride string
+		hostname    model.Hostname
+		subset      string
+		port        *model.Port
+		want        string
+		err         error
+		inbound     bool
 	}{
 		{
 			hostname: "hostname",
 			subset:   "subset",
 			port:     &model.Port{Name: "http", Port: 80, Protocol: model.ProtocolHTTP},
-			want:     "outbound|http|subset|hostname",
+			want:     "_http._subset.hostname",
 		},
 		{
 			hostname: "hostname",
 			subset:   "subset",
 			port:     &model.Port{Port: 80, Protocol: model.ProtocolHTTP},
-			want:     "outbound||subset|hostname",
+			want:     "_80._subset.hostname",
 		},
 		{
 			hostname: "hostname",
 			subset:   "",
 			port:     &model.Port{Name: "http", Port: 80, Protocol: model.ProtocolHTTP},
-			want:     "outbound|http||hostname",
+			want:     "_http._.hostname",
```

Comment:
Can we please have out...? Makes it easy to distinguish stats. And regexes in grafana are easier. For envoy level stats.